PacketTunnelProvider/NetworkProtection/ConfigurationStore.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  ConfigurationStore.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import os.log
+import Core
+import Configuration
+import Persistence
+
+struct ConfigurationStore: ConfigurationStoring {
+
+    enum Error: Swift.Error {
+        case unsupportedConfig
+    }
+
+    private var defaults: KeyValueStoring
+
+    private var privacyConfigurationEtagKey: String {
+        return "privacyConfiguration"
```

Comment:
This key can likely be updated to mention that it's an ETag:

```suggestion
        return "privacyConfigurationEtag"
```

Reply:
This was set to match the main app which has more etags then just remote config. None of those keys mention the etag so if we update this one we should update them all in the main app.

Reply:
No worries, I think we can leave this one in that case - thanks!